app/api/images/images.controller.js

```diff
@@ -403,6 +420,274 @@ exports.getFootprint = utils.route(async (req, res) => {
+
+exports.submitImage = utils.route(async (req, res) => {
+  
+  //CHECKS
+  //check image does not exist already
+  const existingImage = await models.images.findOne({
+    where: {
+      original_id: req.body.original_id,
+      collection_id: req.collection.id
+    }
+  });
+
+  if (existingImage) {
+    throw requestBodyValidationError(req, [
+      {
+        location: 'body',
+        path: '',
+        message: req.__('images.submitted.originalIdAlreadyExist'),
+        validation: 'imageOriginalIdExist'
+      }
+    ])
+  } 
+
+  //check if given photographers already exist and find them
+  let req_photographer_ids = req.body.photographer_ids;
+  if (!req_photographer_ids || req_photographer_ids.length === 0) {
+    req_photographer_ids = [3]; /* anonyme */
+  }
+  //throw error if no photographer with given id is found
+  const res_photographers = await findPhotographers(req, req_photographer_ids)
+
+
+  //CREATE ATTRIBUTES
+
+  //exact date
+  let exact_date_req;
+  if (req.body.date_shot) {
+    exact_date_req = true;
+  } else if (req.body.date_shot_min && req.body.date_shot_max) {
+    exact_date_req = false;
+  } else {
+    //if no date given throw error
+    throw requestBodyValidationError(req, [
+      {
+        location: 'body',
+        path: '',
+        message: req.__('images.submitted.dateRequired'),
+        validation: 'imageDateRequired'
+      }
+    ])
+  }
+
+  //INSERT IMAGE AND APRIORI LOCATION IN DATABASE
+
+  //table images
+  const newImage = await models.images.create({
+    original_id: req.body.original_id,
+    state: 'initial',
+    owner_id: req.collection.owner_id,
+    collection_id: req.collection.id,
+    is_published: req.body.is_published,
+    name: req.body.name,
+    date_inserted: models.sequelize.literal("current_timestamp"),
+    exact_date: exact_date_req,
+    iiif_data: {
+      image_service3_url: req.body.iiif_link
+    },
+    title: req.body.title,
+    orig_title: req.body.title,
+    caption: req.body.caption,
+    orig_caption: req.body.caption,
+    license: req.body.license,
+    download_link: req.body.download_link,
+    link: req.body.link,
+    shop_link: req.body.shop_link,
+    observation_enabled: req.body.observation_enabled,
+    correction_enabled: req.body.correction_enabled,
+    view_type: req.body.view_type,
+    height: req.body.height,
+    width: req.body.width,
+    date_orig: req.body.date_orig,
+    date_shot: req.body.date_shot,
+    date_shot_min: req.body.date_shot_min,
+    date_shot_max: req.body.date_shot_max,
+    apriori_altitude: req.body.apriori_location.altitude,
+  });
+
+  //add association in join table images_photographers
+  newImage.addPhotographer(res_photographers);
+
+  //table apriori_location
+  const req_apriori_location= req.body.apriori_location;
+  const longitude = req_apriori_location.longitude;
+  const latitude = req_apriori_location.latitude;
+  const altitude = req_apriori_location.altitude ? req_apriori_location.altitude : 0;
```

Comment:
probably better to put 1000 or something similar to be sure not to have the camera in the ground